### PR TITLE
Update Readme.

### PR DESCRIPTION
**Pull request by scottdangelo**
Update introduction.
Update Steps.
Add examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,43 @@
-# Feature Extraction and Reverse Image Search on Caltech-101 Dataset
-This notebook will guide you through the process of analyzing an image dataset using a pre-trained convolution network (VGG16) and extracting feature vectors for each image
+# Accelerate Reverse Image Search with GPU for feature extraction
 
-Post analysis we try to demonstrate 'reverse image search' one of the widely popular applications of image analysis.
+In this code pattern, we will guide you through the process of analyzing an image dataset using a pre-trained convolution network (VGG16) and extracting feature vectors for each image using a Jupyter notebook.
+This is a computationally expensive process, requiring over 300 times longer on a CPU versus a GPU. We'll use the GPU environment on Watson Studio or on your local machine to accelerate feature extraction.
```
Comment (rhagarty) [inline README.md]: this first sentence is confusing - do you mean "takes 300 time longer"?

File: README.md [hunk 2]
```diff
@@ -1,27 +1,43 @@
...
+Post analysis we try to demonstrate 'reverse image search' one of the widely popular applications of image analysis. Reverse image search is a content-based image retrieval (CBIR) query technique that involves providing the CBIR system with a sample image that it will then base its search upon; in terms of information retrieval, the sample image is what formulates a search query. In particular, reverse image search is characterized by a lack of search terms.[1]
```
Comment (rhagarty) [inline README.md]: first sentence confusing - maybe needs a comma after "search"?

File: README.md [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
 ## Watch the Video:
 
 [![video](http://img.youtube.com/vi/FpWsLvXFCy0/0.jpg)](https://youtu.be/FpWsLvXFCy0)
 
+## Steps
+
+1. [Clone the repository](#1-clone-the-repository)
+2. Perform either 2a or 2b:  
+  2a. [Create a notebook in IBM Watson Studio](#2a-create-a-notebook-in-ibm-watson-studio)  
+  2b. [Run notebook locally](#2b-run-notebook-locally)
```
Comment (rhagarty) [inline README.md]: This link doesn't work

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rhagarty) [reply README.md]: line 4

File: README.md (hunk 2 above)
Comment (rhagarty) [reply README.md]: line 5

File: README.md (hunk 3 above)
Comment (rhagarty) [reply README.md]: 2b.

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: fixed

**Review by rhagarty — APPROVED** [review]

**Pull request merged**